.stylelintrc.json

```diff
@@ -1,20 +1,20 @@
 {
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your branch name more descriptive such as **feature/countries_data** 👍 Your branch name should tell us what is happening in this branch 😊